src/Objs/ProductCategory/ProductCategoryComponent.tsx

```diff
@@ -67,4 +67,16 @@ const LOCALIZERS = {
     items1: '1 item',
     itemsMany: '__count__ items',
   },
+  fr: {
+    headline: 'Catégorie de produit',
+    items0: 'Aucun article',
+    items1: '1 article',
+    itemsMany: '__count__ articles',
+  },
+  pl: {
+    headline: 'Kategoria produktu',
+    items0: 'Brak produktów',
+    items1: '1 produkt',
+    itemsMany: '__count__ produktów',
```

Comment:
"0 produktów" is correct, but "1 produkt," "2-3-4 produkty," and only 5 and beyond will match the form "produktów" to 21 🤣 because "22 produkty" - requires the modified form once again. Depending on the number before the verb, "produkt" can appear in at least 12 different ways in Polish, if I am right IDK, maybe more. 

I would use "Liczba produktów: count" (Means: Number of products: X) if we wanted to make it more generic.

Reply:
Wow, I didn't know that counting can be so complicated. Thanks for pointing this out!